policy/datasets.yaml

```diff
@@ -1501,6 +1501,20 @@ plotPhotonTransferCurve_metadata:
     storage: YamlStorage
     python: lsst.daf.base.PropertySet
     template: ''
+measureLinearity_config:
+    description: "Configuration for the linearity fitting task."
+    persistable: Config
+    python: lsst.cp.pipe.linearity.MeasureLinearityConfig
+    storage: ConfigStorage
+    tables:
+    - 'raw'
+    template: config/measureLinearity.py
+measureLinearity_metadata:
+    description: "Metadata for the linearity fitting task."
+    persistable: PropertySet
+    storage: YamlStorage
+    python: lsst.daf.base.PropertySet
+    template: ''
```

Comment:
This while change looks fine, but the template is blank (also fine) I just don't see a PR for obs_lsst where I assume this is being run, so just checking how this works, or if Jira has failed to pick up the ticket.

Reply:
Nothing useful is being written to the metadata, so I'd ignored it.  I've added an obs_lsst PR (261) to define the metadata output to prevent future issues.